kolibri/plugins/learn/assets/src/views/thumbnails/ChannelThumbnail.vue

```diff
@@ -10,14 +9,12 @@
       class="image"
       :src="thumbnailUrl"
       alt=""
-    >
+      loading="lazy"
```

Comment:
This looks like the right change, and helpful to actually close the tag when it wasn't before!